vault/changelog.release.2021-11-23.md

```diff
@@ -0,0 +1,100 @@
+---
+id: Ku1qjbDYlsxk4rfMoFEza
+title: "0.70"
+desc: ''
+updated: 1637695774398
+created: 1637687416614
+---
+
+Dendron 0.70 has sprouted  🌱
+
+Dendronites can now use `Dendron: Initialize Workspace` to create [[native workspaces|dendron.topic.workspace.native]]: this means you can get Dendron working in your code repos, following a [Docs-as-Code](https://www.writethedocs.org/guide/docs-as-code/) approach. Jump in and take a look!
```

Comment:
Since native workspaces are still new, would still add a little explanation inline. Generally, the following format works well: `{feature} {explanation} {impact}`
Can we rephrase to the following?

```
Dendronites can now use `Dendron: Initialize Workspace` to create [[native workspaces|dendron.topic.workspace.native]]. This means that you can now use Dendron inside an existing project by turning one of its folders into your Dendron workspace. 
For developers, this takes [Docs-as-Code](https://www.writethedocs.org/guide/docs-as-code/) to a whole new level!
```